tests/serverpod_test_server/test_e2e/service_protocol_test.dart

```diff
@@ -379,8 +379,8 @@ void main() {
         expect(table.foreignKeys.first.referenceTable, 'object_field_scopes');
         expect(table.foreignKeys.first.onUpdate,
             service.ForeignKeyAction.noAction);
-        expect(
-            table.foreignKeys.first.onDelete, service.ForeignKeyAction.cascade);
+        expect(table.foreignKeys.first.onDelete,
```

Comment:
Suggestion: Preserve old behavior for these tests.

Feels like we shouldn't change the behavior of these tests as part of this PR. We have a separate test that validates this in a better way, so maybe we don't need to update this test?

Reply:
I believe this relation is setup with the `relation` keyword, hence we would have to change the test too.

Reply:
Yes, but the introduction of the relation keyword for these tests was also part of the changes for the 1.2 update. Therefore, it feels like we should preserve the behavior/intention of the test unless we want to do an intentional change (Which probably more would be to remove the test and replace it with something in line with our testing strategy).

I guess I'm OK with preserving the change for the test as well. Just want to make sure that we are changing these things intentionally.

Reply:
The full migration in my eyes for these classes from 1.1 to 1.2 is replacing the parent keyword with `relation` but preserving the old behavior.

But since these are test files we are OK with changing them anyway we like.